src/components/axes/abstractAxis.ts

```diff
@@ -25,7 +25,7 @@ export module Axis {
     protected _computedHeight: number;
     private _endTickLength = 5;
     private _tickLength = 5;
-    private _tickLabelPadding = 10;
+    protected _tickLabelPadding = 10;
```

Comment:
Use the public `tickLabelPadding()` method to get and set this value, rather than exposing it as `protected`.